Cargo.toml

```diff
@@ -66,7 +66,7 @@ serde_yaml = "0.9.34"
 strum.workspace = true
 time = { version = "0.3.37", features = ["formatting"] }
 time-humanize = { version = "0.1.3", features = ["time"] }
-tokei = "12.1.2"
+tokei = "13.0.0-alpha.7"
```

Comment:
@o2sh So we're good with moving to a pre-release? I don't have a problem with it, but what are your thoughts?

Reply:
It's definitely not ideal, but we can always roll back if we find any issues

Reply:
Sadly, the upstream release is too slow. It took 3 months from merging this PR to the release. I almost forgot about it.